evalai/challenges.py

```diff
@@ -21,12 +22,19 @@ def challenges(ctx):
 
 @click.group(invoke_without_command=True, name='list')
 @click.pass_context
-def list_challenges(ctx):
+@click.option('-participate', default='false',
+              help="Show the challenges that you've participated")
+@click.option('-host', default='false',
+              help="Show the challenges that you've hosted")
+def list_challenges(ctx, participate, host):
     """
-    Used to list all the challenges.
-    Invoked by running `evalai challenges list`
+    Lists all challenges.
```

Comment:
Why did you undo the docstring?

Reply:
It was an accident while merging, I'll change it back. :D